simtools/visualization/visualize.py

```diff
@@ -65,7 +65,7 @@
     "#A4657A",
     "#424D9C",
     "#DC575A",
-    "#1D2D38",
+    "#1D2d38",
```

Comment:
You should be careful with search and replace...
It doesn't make a difference in this case, but in the future...

Reply:
Yes, it does! I've noted the first, but not the second one! Fixed.